searxstats/fetcher/timing.py

```diff
@@ -157,16 +180,16 @@ async def fetch_one(instance_url: str, detail) -> dict:
 
             # /search instead of / : https://github.com/searx/searx/pull/1681
             search_url = urljoin(instance_url, 'search')
-            theme = 'simple' if detail['generator'] == 'searxng' else 'oscar'
+            theme = 'simple'
             print(search_url, '(', theme, ')')
             check_result = CheckResultByTheme[theme]
-            default_params = {'theme': theme}
+            default_params = {'theme': 'simple'}
```

Comment:
Since all instances are SearXNG forks, we can safely keep only the simple theme.